prov/sm2/src/sm2_progress.c

```diff
@@ -41,6 +41,7 @@
 #include "ofi_mr.h"
 #include "sm2.h"
 #include "sm2_fifo.h"
+#include "sm2_rma.h"
```

Comment:
I think we should put these functions in sm2_rma.c ... what do you think?

Reply:
I wasn't sure which functions you meant here.  You mean alloc and free?  We still need free in progress as well.